SPECS/dracut/dracut.spec

```diff
@@ -1,48 +1,60 @@
-%define dracutlibdir %{_libdir}/dracut
-%define _unitdir %{_libdir}/systemd/system
+%define dracutlibdir        %{_libdir}/%{name}
+%global __requires_exclude  pkg-config
 
 Summary:        dracut to create initramfs
 Name:           dracut
-Version:        055
-Release:        7%{?dist}
+Version:        059
```

Comment:
Can we please attach a TestBuild pipeline. This ensures that ARM64 also works, because mostly folks test only on x86_64.

Reply:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=501029&view=results

Reply:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=501277&view=results

Reply:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=501293&view=results